lms/djangoapps/django_comment_client/forum/views.py

```diff
@@ -129,8 +131,21 @@ def get_threads(request, course_key, discussion_id=None, per_page=THREADS_PER_PA
 
     return threads, query_params
 
+def use_bulk_ops(view_func):
+    """
+    wrap internal request handling inside a modulestore bulk op, significantly
+    reducing redundant database calls.
+    """
+    @wraps(view_func)
+    def wrapped_view(request, course_id, *args, **kwargs):
+        course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
```

Comment:
Why `SlashSeparatedCourseKey.from_deprecated_string` v `CourseKey.from_string`? I think it should be the latter.

As long as you're parsing it here, why not pass the parsed key rather than the string to the wrapped fn? (Less wrong than the other).